cli/main.rs

```diff
@@ -134,7 +135,9 @@ fn create_web_worker_callback(
       args.main_module,
       args.worker_id,
       &options,
-    );
+    )
+    // FIXME(nayeemrmn): Don't unsafely unwrap here.
+    .unwrap();
```

Comment:
Fix before landing.

I've moved things around a bit so this is the only one -- @bartlomieju any ideas on how to return a `Result` in this section?

Reply:
@nayeemrmn you'll have to change type of `CreateWebWorkerCb`, I'm fine with that

Reply:
Done, it's read and stored in `ProgramState`.